.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: FAQ
+    url: https://github.com/OpenRA/OpenRA/wiki/FAQ#common-issues
+    about: FAQ with common questions.
```

Comment:
```suggestion
    about: see frequently asked questions
```

Reply:
Should in any case start with a capital letter. "About" should add some explanation without being too redundant, so maybe something like `Find answers to commonly asked questions.`?

Reply:
If we're going to use "FAQ" at all I think we should use "frequently asked questions" somewhere, which explains it in case someone isn't familiar with 'FAQ'.

Reply:
My two cents:

name: Frequently Asked Questions
about: Explanations for common problems and questions.